pkg/controller/argocd/argocd_controller.go

```diff
@@ -118,10 +95,6 @@ func filterPredicate(assert func(namespace, name string) bool) predicate.Funcs {
 	}
 }
 
-func assertArgoCD(namespace, name string) bool {
-	return namespace == argocdNS && argocdInstanceName == name
-}
-
 func assertArgoCDRoute(namespace, name string) bool {
```

Comment:
nit: I'd have called this `argoCDRouteFilter`

Reply:
The function name is updated